mattermost/mattermost-marketplace
# Ensure manually added plugins stay in the database

#### Summary
Prior to this PR the generator would drop all manually added plugins. These changes ensure the generator only adds new plugins and never removes old ones.
 This also means that removing a GH release does not longer drop the plugin from the Marketplace. I tried preserving the old behaviour, but I didn't found an elegant way. 1/5 that it's not worth fixing this, as that case should rarely happend.

#### QA Steps
1. Run `go run ./cmd/generator/ add mattermost-plugin-todo v0.2.0 --beta --official`
2. Ensure via `git diff` that Todo plugin was added
3. Run `make plugins.json`
4. Ensure via `git diff` that Todo plugin still exists in `plugins.json`

#### Ticket Link
Fixes https://github.com/mattermost/mattermost-marketplace/issues/67
